### PR TITLE
betterdiscord-installer: init at 1.0.0-beta

### PR DESCRIPTION
**Pull request by IvarWithoutBones**
###### Motivation for this change
Seems to be working out of the box for me.

cc @ObserverOfTime

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ObserverOfTime** [comment]
Wouldn't `betterdiscord-installer` or simply `betterdiscord` be a better name?

File: pkgs/tools/misc/betterdiscordinstaller/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{ appimageTools, lib, fetchurl }:
+let
+  pname = "betterdiscordinstaller";
+  version = "1.0.0-beta";
+  name = "BetterDiscord-Linux";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/betterdiscordinstaller/default.nix:5]:
```suggestion
  pname = "betterdiscordinstaller";
  version = "1.0.0-beta";
  name = "${pname}-${version}";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by TheOPtimal** [comment]
I think a better-suited name would be `betterdiscord-installer`, since installing the package doesn't automatically just install it in Discord - you have to run the installer.

**Comment by IvarWithoutBones** [comment]
> I think a better-suited name would be `betterdiscord-installer`, since installing the package doesn't automatically just install it in Discord - you have to run the installer.

I agree, I've changed the packages name to that.

**Pull request merged**